pages/samvera/manager_guide/hyrax_2.1/batch-ops-2.1.md

```diff
@@ -0,0 +1,54 @@
+
+### Adding Files to Works
+The “Files” tab is for uploading files, either locally or from a cloud service. This works in the same way as adding files to a new work. You can upload one file at a time, or select multiple files.
+
+![Upload files](/images/screenshots/create-batch-works-2.png)
+
+After uploading files, you can edit the `Display Label` and `Resource Type` for each Work. Each work can be assigned its own `Display Label` (i.e. Title) and `Resource Type`, but all other metadata will be applied to all works created in the batch. To change the resource type of all files at once, select the desired resource type for the first file and click “Set all to this Resource Type”
+
+![Editing Display Labels and Resource Types](/images/screenshots/create-batch-works-3.png)
+
+### Describing Works
+The `Descriptions` tab asks for `Creator`, `Keyword`, and `Rights Statement`. You also have the option to apply additional description fields. Each description field you apply will be applied to all works in the batch.
+
+![Descriptions](/images/screenshots/create-batch-works-4.png)
+
+### Relationships, Sharing, and Saving
+The  `Relationships` and `Sharing` tabs are identical to the tabs you would see while depositing a work. The process for Saving the batch of works is the same as saving a single work. Every selection you make for these options will be applied to all of the works in the batch.
+
+## Adding Works to a Collection
+As a Repository Administrator or the owner of works, you have the ability to add one or more works to a collection. From your `Dashboard`, click on `Works`, in the sidebar. To the left of each collection title, there is a box that you can click on to select that work. Whenever you select a box, whether it is one or more or all, a red button will appear labelled `Add to collection`. Clicking the button will initiate a dialog box asking if you are sure. This step is irreversible.
+
+## Deleting Works
+As a Repository Administrator or the owner of works, you have the ability to delete works in bulk. From your `Dashboard`, click on `Works`, in the sidebar. To the left of each collection title, there is a box that you can click on to select that work. Whenever you select a box, whether it is one or more or all, a red button will appear labelled `Delete Selected`. Clicking the button will initiate a dialog box asking if you are sure. This step is irreversible.
+
+## Deleting Collections
+As a Repository Administrator, you have the ability to delete collections in bulk. From your `Dashboard`, click on `Collections`, in the sidebar. To the left of each collection title, there is a box that you can click on to select that collection. Whenever you select a box, whether it is one or more or all, a red button will appear labelled `Delete collections`. Clicking the button will initiate a dialog box asking if you are sure. This step is irreversible.
```

Comment:
Add "You can not delete Admin Sets that contain. You must first delete the works or move them to another set".

Reply:
done